build.gradle

```diff
@@ -0,0 +1,68 @@
+plugins {
+    id 'java-gradle-plugin'
+    id 'jacoco'
+    id 'com.gradle.plugin-publish' version '0.9.7'
+    id 'com.github.hierynomus.license' version '0.13.1'
+    id 'org.sonarqube' version '2.2.1'
+}
+
+repositories {
+   jcenter()
+   mavenLocal()
+}
+
+version = '0.0.1'
+group = 'com.github.kaklakariada'
+
+ext {
+  gradlePluginId = 'com.github.kaklakariada.openfasttrack'
+}
+
+dependencies {
+    compile 'com.github.kaklakariada:openfasttrack:0.1.1-alpha'
```

Comment:
This requires applying for a new group id at bintray.com. Please select a group id and tell me.